chall03/gmolin.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"net/http"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+)
+
+func http_get_hex(id string, hex string)string{
+	result, err := http.Get("https://chall03.hive.fi/?id="+id+"&resp="+hex)
+	if err != nil {
+		fmt.Println(err)
+	}
+	body, err := ioutil.ReadAll(result.Body)
+	if err != nil{
+		fmt.Println(err)
+	}
+	return string(body)
+}
+
+func http_get_id_rgb(www string)[]string{
+	result, err := http.Get(www)
+	if err != nil {
+		fmt.Println(err)
+	}
+	body, err := ioutil.ReadAll(result.Body)
+	if err != nil{
+		fmt.Println(err)
+	}
```

Comment:
Somehow, your both function are pretty doing the same thing, I would recommend not creating another function to do that but let the "main" logic into your main.

If tomorrow you are asked to use the `result` variable, you will have to pass/return parameters when it could be directly used in your main scope if it's declared there. (You may be asked to check a `resp.StatusCode`, for example, or `resp.Header()`